lib/torrent.js

```diff
@@ -73,6 +73,8 @@ class Torrent extends EventEmitter {
     this._store = opts.store || FSChunkStore
     this._getAnnounceOpts = opts.getAnnounceOpts
 
+    if (typeof opts.private === 'boolean') this.private = opts.private
```

Comment:
could use a comment saying that `this.private === undefined` when it wasn't passed as an option and otherwise it will overwrite the private field in the torrent file

Otherwise it just looks like you're making sure the input is a boolean instead of relying on it being undefined later

Reply:
Agreed